pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -185,7 +185,13 @@ func getV1Registries(config *tomlConfig) ([]Registry, error) {
 	getRegistry := func(s string) (*Registry, error) { // Note: _pointer_ to a long-lived object
 		url, err := parseURL(s)
 		if err != nil {
-			return nil, err
+			// fallback to https to be compatible with v1
+			if strings.Contains(err.Error(), "unspecified URI scheme: ") {
```

Comment:
I considered that, but I chose the current solution as I think that special casing `parseURL` will be more expensive. Golang doesn't support default values, so we had to always do checks in comparison to the upper error case that only appears if a given `registries.conf` is written in the v1 format.

Reply:
(Depending on https://github.com/projectatomic/libpod/issues/721 , this may end up completely differently.)

> I considered that, but I chose the current solution as I think that special casing parseURL will be more expensive.

Not passing a single boolean parameter at the cost of adding a compiler-invisible aspect to the interface of `parseURL` doesn’t seem all that worthwhile to me.   But you did add a test, so this is not going to be silently broken; fair enough.

Please rebase and I can merge this.

Reply:
Done. Thanks a lot, @mtrmac!